switchapi/src/switch_init.c

```diff
@@ -446,12 +446,13 @@ switch_api_init_default_acl_entries(switch_device_t device)
     acl_kvp[1].mask.u.mask = 0xFFFFFFFF;
     memset(&action_params, 0, sizeof(switch_acl_action_params_t));
     action_params.cpu_redirect.reason_code = 0;
-    switch_api_acl_rule_create(device, acl_handle, priority++, 2,
+    switch_api_acl_rule_create(device, acl_handle, 
+                               2000, /* keep priority lower than reason-codes */
```

Comment:
Maybe we should have a predefined reason code for this in switch_hostif.c as well.